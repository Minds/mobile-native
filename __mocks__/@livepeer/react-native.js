// mock @livepeer/react-native
export const LivepeerConfig = {
  setConfig: jest.fn(),
  getConfig: jest.fn(),
  get: jest.fn(),
  set: jest.fn(),
};

export const createReactClient = jest.fn();
export const createLivepeerClient = jest.fn();
export const Player = jest.fn();
