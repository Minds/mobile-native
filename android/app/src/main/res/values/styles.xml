<resources>

    <!-- Base application theme. -->
    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
        <item name="android:textColor">#000000</item>
        <item name="android:forceDarkAllowed">false</item>
        <item name="android:editTextBackground">@drawable/rn_edit_text_material</item>
        <!-- Customize your theme here. -->
    </style>

    <style name="BootTheme" parent="AppTheme">
        <!-- set bootsplash.xml as activity background -->
        <item name="android:background">@drawable/bootsplash</item>
    </style>


</resources>
