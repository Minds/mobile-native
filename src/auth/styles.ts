import { StyleSheet, Dimensions, Platform } from 'react-native';
const { width } = Dimensions.get('window');

export const styles = StyleSheet.create({
  inputBackground: {
    backgroundColor: 'rgba(0,0,0,0.25)',
    borderColor: 'rgba(255,255,255,0.30)',
  },
  shadow: {
    backgroundColor: '#00000000',
    shadowColor: '#000',
    shadowOffset: {
      width: 0,
      height: 0,
    },
    shadowOpacity: 0.8,
    shadowRadius: 20,
    elevation: 6,
    overflow: 'visible',
    zIndex: 999,
  },
  lightButton: {
    borderColor: 'rgba(255,255,255,0.20)',
  },
});

export const shadowOpt = {
  width,
  height: 170,
  color: '#000',
  border: 20,
  opacity: 0.2,
  x: 0,
  y: 0,
};

export const icon = { top: Platform.OS === 'ios' ? 26 : 28 };
