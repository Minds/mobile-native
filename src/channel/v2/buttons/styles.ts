import { Platform, StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
  button: {
    padding: Platform.select({ ios: 8, android: 6 }),
    marginLeft: 5,
    shadowOpacity: 0.25,
    shadowRadius: 2,
    shadowOffset: { width: 0, height: 2 },
    shadowColor: '#000',
  },
});
