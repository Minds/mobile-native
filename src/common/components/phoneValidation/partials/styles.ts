import { StyleSheet } from 'react-native';

export const style = StyleSheet.create({
  view: {
    backgroundColor: '#fff',
    flex: 1,
    padding: 10,
  },
  cols: {
    flexDirection: 'row',
    marginBottom: 10,
    alignItems: 'stretch',
  },
  col: {
    flexBasis: 0,
    flexGrow: 1,
    marginLeft: 10,
  },
  colFirst: {
    marginLeft: 0,
  },
  form: {
    marginTop: 20,
  },
  phoneInput: {
    borderWidth: 1,
    borderRadius: 3,
    paddingLeft: 10,
    paddingRight: 10,
  },
  phoneTextInput: {
    fontSize: 16,
    fontWeight: '700',
  },
});
