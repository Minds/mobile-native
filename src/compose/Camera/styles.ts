import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
  icon: {
    padding: 10,
    color: 'white',
    textShadowColor: 'rgba(0, 0, 0, 0.35)',
    textShadowOffset: { width: -1, height: 1 },
    textShadowRadius: 2.22,
  },
  autoIcon: {
    position: 'absolute',
  },
  galleryIcon: {
    padding: 10,
    color: 'white',
    alignSelf: 'center',
    marginLeft: 15,
    textShadowColor: 'rgba(0, 0, 0, 0.35)',
    textShadowOffset: { width: -1, height: 1 },
    textShadowRadius: 2.22,
  },
});
