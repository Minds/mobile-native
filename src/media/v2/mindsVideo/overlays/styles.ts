import { StyleSheet } from 'react-native';
import Colors from '../../../../styles/Colors';

export const styles = StyleSheet.create({
  overlayContainer: {
    position: 'absolute',
    top: 0,
    left: 0,
    right: 0,
    bottom: 0,
    flexDirection: 'column',
    justifyContent: 'center',
    alignItems: 'center',
  },
  overlayContainerTransparent: {
    backgroundColor: 'transparent',
  },
  errorText: {
    color: Colors.darkGreyed,
  },
  videoIcon: {
    position: 'relative',
    alignSelf: 'center',
    bottom: 0,
    left: 0,
    right: 0,
    top: 0,
  },
  textShadow: {
    textShadowColor: 'rgba(0, 0, 0, 0.45)',
    textShadowOffset: { width: 1, height: 1 },
    textShadowRadius: 1,
  },
  controlSettingsContainer: {
    position: 'absolute',
    top: 10,
    right: 0,
    margin: 8,
    paddingRight: 5,
  },
  controlBarContainer: {
    flexDirection: 'row',
    position: 'absolute',
    bottom: 0,
    left: 0,
    right: 0,
    alignItems: 'center',
    paddingLeft: 4,
    paddingRight: 4,
    borderRadius: 3,
    backgroundColor: 'rgba(48,48,48,0.7)',
  },
});
