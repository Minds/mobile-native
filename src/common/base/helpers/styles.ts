import ThemedStyles from '~/styles/ThemedStyles';
import { ColorsNameType } from '~/styles/Colors';

export const getPropStyles = (props: any) => {
  const styles: any = [];
  if (!(props && typeof props === 'object')) {
    return styles;
  }

  const keys = Object.keys(props);

  for (let i = 0; i < keys.length; i += 1) {
    const key = keys[i];
    const label = `${key}${props[key]}`;
    try {
      const style = ThemedStyles.style[label];
      if (style) {
        styles.push(style);
      }
    } catch (e) {
      //
    }
  }

  return styles;
};

export const getPropColor = (color: ColorsNameType, active: boolean) => {
  if (active === true) {
    return ThemedStyles.style.colorLink.color;
  }
  if (typeof color !== 'string') {
    return '';
  }
  return color
    ? color.match(/^#/)
      ? color
      : ThemedStyles.getColor(color)
    : null;
};
