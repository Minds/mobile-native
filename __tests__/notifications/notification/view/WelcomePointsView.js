import 'react-native';
import React from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import WelcomePointsView from '../../../../src/notifications/notification/view/WelcomePointsView';
import styles from '../../../../src/notifications/notification/style';

// fake data generation
import boostNotificationFactory from '../../../../__mocks__/fake/notifications/BoostNotificationFactory';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const entity = boostNotificationFactory('tag', 'comment');
  const notification = renderer
    .create(<WelcomePointsView styles={styles} entity={entity} />)
    .toJSON();

  expect(notification).toMatchSnapshot();
});
