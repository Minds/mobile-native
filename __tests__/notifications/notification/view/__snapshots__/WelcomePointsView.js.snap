// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onPress={[Function]}
  style={
    Object {
      "flex": 1,
      "marginLeft": 8,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "flexWrap": "wrap",
      }
    }
  >
    <Text>
      Welcome to Minds. We have given you
       
      <Text
        style={
          Object {
            "fontWeight": "600",
          }
        }
      >
        100 points
      </Text>
       to get you started.
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontSize": 11,
      }
    }
  >
    Invalid date
  </Text>
</View>
`;
